recipes/logrotate/logrotate_3.9.2.oe

```diff
@@ -0,0 +1,30 @@
+DESCRIPTION = "The logrotate utility is designed to simplify the administration of log files on a system which generates a lot of log files."
+HOMEPAGE = "https://github.com/logrotate/logrotate"
+LICENSE = "GPL-2.0"
+
+COMPATIBLE_HOST_ARCHS = ".*linux"
+
+SRC_URI = "https://github.com/logrotate/logrotate/archive/${PV}.tar.gz" 
+
+DEPENDS =+ "libpopt"
```

Comment:
acl support is encapsulated inside a condition that checks for availability of acl. As it isn't available it should default to --with-acl=no. The same goes for selinux.

We could add a flag for setting --with-acl however...but currently the recipe builds a minimal logrotate.